src/transport_picoquic.h

```diff
@@ -132,7 +132,7 @@ class PicoQuicTransport : public ITransport
 
         DataHeader data_header {};                           /// Data header to use for start of stream or for every datagram
 
-        uint64_t current_stream_id {0};                      /// Current active stream if the value is >= 4
+        uint64_t current_stream_id {~(uint64_t(0))};         /// Current active stream if the value is >= 4
```

Comment:
can we make it std::optional<uint64_t> instead ?

Reply:
Sure, I'll make that change.